apps/web/cypress/integration/partthree.spec.js

```diff
@@ -0,0 +1,26 @@
+describe('load page.com', () => {
+  beforeEach(() => {
+    cy.visit('/submission/1');
+    // Fill out first page
+    cy.get(`[data-cy=firstname]`).type('testname').should('have.value', 'testname');
```

Comment:
This duplication is weird, when we change a field we need to update multiple tests right? Not sure what the best implementation here is but I think if we end up needing all of this duplication we should just make it a single test

Reply:
It's a bit jank, but I don't see how to access pages further in the process when we have a redirect in effect. It may end up that we have to have this all in one very large final test.

Reply:
I'll definitely remove all the 'should' calls here. We don't need to test anything in the before step

Reply:
I've disabled the form redirection when `NODE_ENV === test` in the past, but I think that's an antipattern. I think I'd like to see this as a single test